code/datums/outfits/jobs/ghostroles.dm

```diff
@@ -58,22 +59,25 @@
 	head = /obj/item/clothing/head/soft/sarge2soft
 	back = /obj/item/storage/backpack/security
 	backpack_contents = list(/obj/item/ammo_magazine/rifle_75_drum = 3, /obj/item/ammo_magazine/rifle_75_drum/highvelocity = 2, /obj/item/ammo_casing/grenade/flash = 2, /obj/item/ammo_casing/grenade/blast = 2, /obj/item/modular_computer/tablet/lease/preset/command = 1 )
+	id_slot = slot_wear_id
+	id_type =  /obj/item/card/id/syndicate/ert
```

Comment:
marshal ERT already has an ID and its already AA

Reply:
Please read why the IDs were changed in the changelog.